src/http.rs

```diff
@@ -256,7 +261,8 @@ impl HttpClient {
             .await?
             .text()
             .await?;
-        TxLogResponse::deserialize(resp)
+
+        Ok(Deserialize::deserialize(&Edn::from_str(&resp)?)?)
```

Comment:
Lol, I just realized this can be simplified for:

```rust
Ok(edn_rs::from_str(&resp)?)
```

I forgot that I've made `edn_rs::from_str` just for this hahahah :sweat_smile:

Reply:
can't we just return `edn_rs::from_str(&resp)` then?

Reply:
Same problem, I will put the `map_err(|e| e.into())` since you prefer it :+1:

Reply:
don't think it should be what someone prefers, but what wseems cleaner and more community readable. What do you think? 

I get confused reading `Ok(...?)`

Can't we do something like:
```
Edn::from_str(&resp).map(Deserialize::deserialize)
  .map_err(|e| e.into())
```

Reply:
I transformed this into:
```rust
edn_rs::from_str(&resp)
    .map_err(|e| e.into())
```